src/ChirpCore/Domain/Cheep.cs

```diff
@@ -11,7 +11,8 @@ public class Cheep
 		public int CheepId { get; set; }
 		public int AuthorId { get; set; }
 		public required Author Author { get; set; }
-		[StringLength(160)] public required string Text { get; set; }
+		[StringLength(160, MinimumLength = 1)]
```

Comment:
[Required] [StringLength(160)]

Reply:
Does not need to add required in the DTO :)